gml_application_schema_toolbox/core/load_gmlas_in_qgis.py

```diff
@@ -99,7 +99,9 @@ def createLayerTreeModelLegendNodes(self, layer_tree_layer):
         return [QgsSimpleLegendNode(layer_tree_layer, self.text, self.icon, self)]
 
 
-def import_in_qgis(gmlas_uri: str, provider: str, schema: Union[str, None] = None):
+def import_in_qgis(
+    gmlas_uri: str, provider: str, add_form_code: bool, schema: Union[str, None] = None
```

Comment:
Can you please provide a description of `add_form_code` parameter in docstring ?